tests/qos/qos_sai_base.py

```diff
@@ -639,21 +646,21 @@ def updateDockerService(host, docker="", action="", service=""):
         for service in services:
             updateDockerService(duthost, action="stop", **service)
 
-        """ Disable linkmgr """
+        """ Stop mux container for dual ToR """
         if 'dualtor' in tbinfo['topo']['name']:
-            duthost.shell('sudo config feature state mux disabled')
-            logger.info("Disable linkmgr for dual ToR testbed")
+            duthost.shell('sudo systemctl stop mux')
```

Comment:
Does the config command have any issue? I thought it does the same

Reply:
I was still seeing some background traffic with older command. With newer one, i don't see anything.